addons/website/views/snippets/s_product_list.xml

```diff
@@ -3,62 +3,93 @@
 
 <template name="Items" id="s_product_list">
     <t t-set="url" value="url or '#'"/>
-    <section class="s_product_list">
-        <div class="container-fluid">
+    <section class="s_product_list pt64 pb64" data-vcss="001">
+        <div class="container">
+            <h3>Our finest selection</h3>
```

Comment:
I'd place the title in a column. This way, the user could easily customize the spacing and alignment.

```suggestion
<div class="col-lg-12">
   <h3>Our finest selection</h3>
</div>
```